### PR TITLE
docs: update code in Decorators.md

### PR DESCRIPTION
**Pull request by shimks**
## Checklist

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `packages/example-*` were updated

### CODE REVIEW

File: docs/site/Decorators.md [hunk 1]
```diff
@@ -72,59 +76,63 @@ controller. For example:
...
 
 A more detailed explanation can be found in
-[Specifying Controller APIs](controllers.htm#specifying-controller-apis)
+[Specifying Controller APIs](Controllers.md#specifying-controller-apis)
 
 ### Operation Decorator
 
 Syntax:
-[`@operation(verb: string, path: string, spec?: OperationObject)`](http://apidocs.loopback.io/@loopback%2fcore/#818)
+[`@operation(verb: string, path: string, spec?: OperationObject)`](https://github.com/strongloop/loopback-next/blob/d6a7d05122280f36d582365aeaa58e80fe31ad8f/packages/openapi-v3/src/operation-decorator.ts#L79)
```
Comment (b-admike) [inline docs/site/Decorators.md]: What are the benefits of linking to the source code as opposed to linking to the `apidocs`?

File: docs/site/Decorators.md [hunk 2]
```diff
@@ -421,62 +426,31 @@ The format of property definitions can be found in
...
-The injection example can be found in
-[Repository#controller-configuration](Repositories.md#controller-configuration)
+The injection example can be found in [Repository#controller-configuration](Repositories.md#controller-configuration)
 
 To create a repository in a controller, you can define your model and datasource
 first, then import them in your controller file:
 
-_To learn more about creating model and datasource, please see the example in
-[Thinking in LoopBack](Thinking-in-LoopBack.htm#define-product-model-repository-and-data-source)_
+*To learn more about creating model and datasource, please see the example in [Best Practices with LoopBack](Implementing-features.md#define-product-model-repository-and-data-source)*
```
Comment (b-admike) [inline docs/site/Decorators.md]: `a model and datasource`

File: docs/site/Decorators.md [hunk 3]
```diff
@@ -421,62 +426,31 @@ The format of property definitions can be found in
...
-Register a specific relation
-
 ### Repository Decorator
 
 Syntax:
-
-```ts
-@repository(model: string | typeof Entity, dataSource?: string | juggler.DataSource)
-```
+`@repository(model: string | typeof Entity, dataSource?: string | juggler.DataSource)`
```
Comment (b-admike) [inline docs/site/Decorators.md]: should we link to source code/apidocs here as well?

File: docs/site/Decorators.md (hunk 1 above)
Comment (shimks) [reply docs/site/Decorators.md]: I don't remember honestly lol. I'll switch it back to apidocs

File: docs/site/Decorators.md (hunk 3 above)
Comment (shimks) [reply docs/site/Decorators.md]: done

File: docs/site/Decorators.md (hunk 1 above)
Comment (b-admike) [reply docs/site/Decorators.md]: they're more visually appealing IMO 🖌

File: docs/site/Decorators.md [hunk 4]
```diff
@@ -340,74 +336,83 @@ Here's an example using 'BasicStrategy': to authenticate user in function
...
+complete with CRUD operations for accessing the model's data.
+A use case can be found in section [Repository decorator](#repository-decorator)
 
-The other one is defining your own repository without using legacy juggler, and
-use an ORM/ODM of your choice.
+The other one is defining your own repository without using legacy juggler,
+and use an ORM/ODM of your choice.
 
 ```ts
 // Missing example here
```
Comment (b-admike) [inline docs/site/Decorators.md]: It feels weird that this comment is split into two lines. feel free to ignore though.

File: docs/site/Decorators.md [hunk 5]
```diff
@@ -255,12 +242,17 @@ dependencies:
 Syntax: `@inject.getter(bindingKey: string)`.
```
Comment (b-admike) [inline docs/site/Decorators.md]: link to https://apidocs.strongloop.com/@loopback%2fcontext/#35 to be consistent?

**Comment by shimks** [comment]
@b-admike I missed some of the links to apidocs out because they're rather bad or they don't exist because of the namespace issue with tsdocs. What do you think is a better alternative?

**Comment by b-admike** [comment]
Oh I thought we resolved the namespace issue a while back (cc @raymondfeng). I don't mind linking all or some to source code too now that I think about it. I slightly prefer linking some to source code, but maybe others have more ideas or strong opinions.

**Comment by shimks** [comment]
@slnode test please

**Pull request merged**